crates/sui-telemetry/src/lib.rs

```diff
@@ -50,6 +71,18 @@ pub async fn send_telemetry_event() {
     send_telemetry_event_impl(telemetry_payload).await
 }
 
+fn get_git_rev() -> String {
+    let output = Command::new("git")
+        .args(&["describe", "--always", "--dirty"])
+        .output()
+        .unwrap();
```

Comment:
we shouldn't unwrap here. If `git` is not installed it will panic and cause the tokio task to fail and exit.

Reply:
nice catch, thanks!